test/cpp/rpc/test_wire_serialization.cpp

```diff
@@ -68,6 +68,13 @@ TEST(WireSerialize, CloneSparseTensors) {
   EXPECT_TRUE(v3.get(0).is_same(sparse));
 }
 
+TEST(WireSerialize, Errors) {
+  EXPECT_THROW((void)torch::distributed::rpc::wireDeserialize("", 0), std::runtime_error);
+  EXPECT_THROW((void)torch::distributed::rpc::wireDeserialize(" ", 1), std::runtime_error);
```

Comment:
Shall we also check the error message here? "failed parse" vs "failed bounds"?

Reply:
sure, just uploaded a version with this check